scripts/update_static_files.py

```diff
@@ -59,6 +60,10 @@ def run():
 
             remote_file_content = remote_file_request.text
             local_file_path = REPO_ROOT_DIR / Path(relative_file_path)
+            local_parent_dir = local_file_path.parent
+
+            if not local_parent_dir.exists():
+                os.mkdir(local_parent_dir)
```

Comment:
```suggestion
                local_parent_dir.mkdir(parents=True)
```
Should work and also deal with missing parents of parents, etc. if we ever add nested stuff